packages/fxa-payments-server/src/routes/Checkout/index.scss

```diff
@@ -1,3 +1,103 @@
-#checkout-route-container {
-    border: 1px solid black;
+@import '../../../../fxa-content-server/app/styles/variables';
+@import '../../../../fxa-content-server/app/styles/breakpoints';
+@import '../../styles/variables';
+
+.subscription-title {
+  padding-top: 20px;
+}
+
+.assurance-copy {
+  justify-content: left;
+}
+
+.step-header {
+  margin-top: 40px;
+}
+
+.accepted-cards {
+  justify-content: center;
+}
+
+.subscription-title + .product-payment {
+  border-top: 0;
+  border-radius: 0 0 8px 8px;
+  box-shadow: 0 -1px white, 0 1px white, -2px 2px 2px -2px rgba(12, 12, 13, 0.1), 2px 2px 2px -2px rgba(12, 12, 13, 0.1);
+
+  @include max-width('tablet') {
+    box-shadow: 0px 1px 4px 0px rgba(12, 12, 13, 0.1);
+    border-radius: 8px;
+  }
+}
+
+.subscription-title + .product-payment {
+  grid-row: 2/3;
+
+  @include max-width('tablet') {
+    grid-row: 3/4;
+  }
+}
+
+.product-payment {
+  grid-row: 1/3;
+  background: $color-white;
+  box-shadow: 0px 1px 4px 0px rgba(12, 12, 13, 0.1);
+  border-radius: 8px;
+  color: rgba(12, 12, 13, 0.8);
+  padding: 16px 16px 60px;
+
+  @include max-width('tablet') {
+    grid-row: 3/4;
+    margin: 24px 16px;
+    min-height: 100%;
+  }
+
+  @include min-width('desktop') {
+    padding: 16px 48px 48px;
+  }
+
+  header {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin-bottom: initial;
+    margin-top: 20px;
+    padding-bottom: 32px;
+    text-align: center;
+
+    @include min-width('desktop') {
+      margin-top: 32px;
+    }
+
+    p {
+      color: #6d6d6e;
+    }
+  }
+
+  .pay-with-heading {
+    color: rgb(115, 115, 115);
+    margin: 0 0 24px 0;
+    display: grid;
+    width: 100%;
+    align-items: center;
+    text-align: center;
+    grid-template-columns: minmax(20px, 1fr) auto minmax(20px, 1fr);
+    grid-gap: 20px;
+    &:before,
+    &:after {
+      content: '';
+      border-top: 1px solid #e0e0e6;
+    }
+
+    @include max-width('tablet') {
+      margin-top: 16px;
+    }
+  }
+
+  .subscription-create-pay-with-other .paypal-button {
+    width: 240px;
+    height: 48px + 48;
```

Comment:
Is this the same as having `96px`?

Reply:
yeah not sure why it was written this way, copied from `subscriptionCreate/index.scss`. happy to change it